### PR TITLE
Added the LinkUpdated activity in the space context

### PR DESCRIPTION
**Pull request by 2403905**
## Related Issue

- Fixes [#10012](https://github.com/owncloud/ocis/issues/10012)

## Motivation and Context

Added the LinkUpdated activity in the space context

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: changelog/unreleased/fix-acitivity-update-public-link.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Bugfix: Added LinkUpdated activity
+
+Added the LinkUpdated activity in the space context
+
+https://github.com/owncloud/ocis/pull/9856
```
Comment (fschade) [inline changelog/unreleased/fix-acitivity-update-public-link.md:5]: .../issues/...

**Review by fschade — APPROVED** [review]
one tiny thing inside the changelog, beside that LGTM

**Review by kobergj — APPROVED** [review]

**Review by dragonchaser — CHANGES_REQUESTED** [review]
Please do not merge yet, we have changes to the same file and i10 adaptions....

**Review by dragonchaser — APPROVED** [review]
merge, the other one needs adaptions to web

**Pull request merged**